README.md

```diff
@@ -36,18 +39,26 @@ The following controls are bound in-game, for gameplay and testing.
 | E | Interact with test cubes |
 | Tab | Toggle scoreboard |
 
-## Contributions
+#### Give us feedback
+We have released the GDK for Unreal this early in development because we want your feedback. Please come and talk to us about the software and the documentation via: [Discord](https://discordapp.com/channels/311273633307951114/339471548647866368) - [Forums](https://forums.improbable.io/) - [GitHub issues in this repository](https://github.com/spatialos/UnrealGDK/issues).
 
-**Feedback**<br/>
-Please give us your feedback! Please see the Unreal GDK [feedback section](https://github.com/improbable/UnrealGDK#give-us-feedback) for more information.
+#### Where to get the GDK and example projects
+The GDK and its starter projects are available on GitHub.
```

Comment:
Do we want to say "starter projects" here? Could be confusing given that one of them is actually called "Starter Project"! Change to "example projects" (like in the heading)?